Sources/PubNub/Networking/Response/ResponseOperator.swift

```diff
@@ -81,12 +81,23 @@ public extension ResponseDecoder {
       GenericServicePayloadResponse.self,
       from: data
     )
+
+    let fullDetails: [ErrorDetail]? = if let payload = generalErrorPayload {
+      [ErrorDetail(
+        message: payload.message.rawValue,
```

Comment:
Sometimes error message stored inside of `error` key. But our newest versions of endpoints stick to the new format.

Reply:
Yes, you're right. [Here's](https://github.com/pubnub/swift/blob/master/Sources/PubNub/Networking/Response/GenericServicePayloadResponse.swift#L250) the full handling for different error message response structures. And all in all, it's parsed into `EndpointResponseMessage`